terracotta-kit/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>terracotta-core-root</artifactId>
+        <groupId>org.terracotta.internal</groupId>
+        <version>5.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>terracotta-kit</artifactId>
+    <packaging>pom</packaging>
+    <name>Terracotta Kit</name>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>org.terracotta</groupId>
+            <artifactId>terracotta</artifactId>
```

Comment:
yeah, this dependency is causing the build failure. 

This particular module is part of integration-root and used by forge plugin to copy server dependencies into kit. we have few options 

1. try using mvn dependency plugin to copy dependencies 
2. create a terracotta module in OSS
3. modify forge plugin

I will try these option in order.